content/news/2022-01-06-bevy-0.6/index.md

```diff
@@ -0,0 +1,1007 @@
+### Directional Shadows
+
+<div class="release-feature-authors">authors: Rob Swain (@superdump)</div>
+
+Directional Lights can now cast "directional shadows", which are "sun-like" shadows cast from a light source infinitely far away. These can be enabled by setting `DirectionalLight::shadows_enabled` to `true`.
+
+![directional light](directional_light.png)
+
+Note: directional shadows currently require more manual configuration than necessary (ex: manual configuration of the shadow projection). We will soon make this more automatic by generating the projection via a combination of simpler settings and fitting the projection to the scene.
+
+### Point Light Shadows
+
+<div class="release-feature-authors">authors: @mtsr, Rob Swain (@superdump), @cart</div>
+
+Point lights can now cast "omnidirectional shadows", which can be enabled by setting `PointLight::shadows_enabled` to `true`:
+
+
+![point light](point_light.png)
+
+### Enabling and Disabling Entity Shadows
+
+<div class="release-feature-authors">authors: Rob Swain (@superdump)</div>
+
+Mesh entities can opt out of casting shadows by adding the {{rust_type(type="struct" crate="bevy_pbr" version="0.6.0" name="NotShadowCaster")}} component.
+
+```rust
+commands.entity(entity).insert(NotShadowCaster);
+```
+
+Likewise they can opt out of receiving shadows by adding the {{rust_type(type="struct" crate="bevy_pbr" version="0.6.0" name="NotShadowReceiver")}} component.
+
+```rust
+commands.entity(entity).insert(NotShadowReceiver);
+```
+
+### Spherical Area Lights
+
+<div class="release-feature-authors">authors: @Josh015</div>
+
+`PointLight` Components can now define a `radius` value, which controls the size of the sphere that emits light. A normal zero-sized "point light" has a radius of zero.
+
+![spherical area lights](spherical_area_lights.png)
+
+(Note that lights with a radius don't normally take up physical space in the world ... I added meshes to help illustrate light position and size)
+
+### Configurable Alpha Blend Modes
+
+<div class="release-feature-authors">authors: Rob Swain (@superdump)</div>
+
+Bevy's StandardMaterial now has a `blend_mode` field, which can be set to `AlphaMode::Opaque`, `AlphaMode::Mask(f32)`, or `AlphaMode::Blend`. This field is properly set when loading GLTF scenes.
+
+![alpha blend modes](alpha_blend.png)
+
+### Clustered Forward Rendering 
+
+<div class="release-feature-authors">authors: Rob Swain (@superdump)</div>
+
+Modern scenes often have many point lights. But when rendering scenes with hundreds of lights, calculating lighting for each light, for each rendered fragment can be prohibitively expensive. Clustered Forward Rendering is a popular approach that increases the number of lights you can have in a scene by dividing up the view frustum into "clusters". Each cluster is then assigned lights based on whether or not they can affect that cluster. This is a form of "culling" that enables fragments to ignore lights that aren't assigned to their cluster.
```

Comment:
Do we have a good link to some reading on CFR? This seems like the sort of topic that would be interesting to dig into for some of the audience.

Reply:
Added the best link I have!